_targets.R

```diff
@@ -49,14 +49,19 @@ site_tp_select <- c("ST","ST-CA","SP")
 # sites 01412350, 01484272, and 01482695 coded as site type "ST" but appear to be tidally-influenced
 omit_nwis_sites <- c("01412350","01484272","01482695") 
 
+# Define PRMS mainstem segments thought to be tidally-influenced (used for filtering sites used in daily/sub-daily models)
+mainstem_reaches_tidal <- c("2771_1","2769_1","2768_1","2767_1","2764_1","2762_1","2763_1","2759_1","2757_1","2752_1",
+                            "2753_1","2755_1","2772_1","388_1","389_1","385_1","386_1","383_1","382_1","377_1","378_1",
+                            "376_1","351_1","344_1","346_1","333_1")
+
```

Comment:
This is fine for now, but we could also use the reach network connectivity table to select the reaches downstream of the mainstem Trenton reach. `reach_attributes_drb.csv` from [ScienceBase](https://www.sciencebase.gov/catalog/item/5f6a289982ce38aaa2449135). I can make an issue to create targets for the datasets in that ScienceBase item because we'll use them for modeling and add to the reach attribute tables for this work

Reply:
OK, good idea. We've downloaded those reach attributes within the DO pipeline, so we could easily port here as well (e.g. https://github.com/USGS-R/drb-do-ml/pull/37).

Reply:
I've added issue #53 to create targets for the network attributes, including the network connectivity matrix.